codegen/src/main/scala/com/soundcloud/twinagle/codegen/Twinagle.scala

```diff
@@ -7,14 +7,23 @@ import sbt.plugins.JvmPlugin
 import sbtprotoc.ProtocPlugin.autoImport.PB
 
 object Twinagle extends AutoPlugin {
+  val scalapbCodeGeneratorOptions = settingKey[Set[scalapb.GeneratorOption]]("Settings for scalapb code generation")
+
   override def requires: Plugins = sbtprotoc.ProtocPlugin && JvmPlugin
 
   override def trigger: PluginTrigger = NoTrigger
 
   override def projectSettings: Seq[Def.Setting[_]] = List(
+    scalapbCodeGeneratorOptions := Set(
+      scalapb.GeneratorOption.FlatPackage // don't include proto filename in scala package name
+    ),
     Compile / PB.targets := Seq(
-      Target(scalapb.gen(grpc = false), (sourceManaged in Compile).value),
-      Target(JvmGenerator("scala-twinagle", ServerClientCodeGenerator), (sourceManaged in Compile).value)
+      Target(scalapb.gen(scalapbCodeGeneratorOptions.value), (sourceManaged in Compile).value),
```

Comment:
we need to make sure that `grpc = false` is always there, otherwise it will actually generate the GRPC classes, won't it?

Reply:
Agreed.

Here are the other defaults:
https://scalapb.github.io/sbt-settings.html#additional-options-to-the-generator

Reply:
I considered enforcing `grpc = false`, but otoh you'd have to go out of your want to turn it on, and if somebody does, I guess they know what they're doing :)

Reply:
@oberkowitz I force-removed the Grpc option now. :+1: ?